dts/bindings/rtc/epson,rx8130ce.yaml

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2024 Måns Ansgariusson <mansgariusson@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+description: RX8130CE RTC
+
+compatible: "epson,rx8130ce-rtc"
+include: i2c-device.yaml
+properties:
+  irq-gpios:
+    type: phandle-array
+
+  clockout-frequency:
+    type: int
+    default: 0
+    description: |
+      Specify the ociillator frequency, the default is 0 (off)
+      - 0 # off
+      - 1 # 1 Hz oscillator
+      - 2 # 1 kHz oscillator
+      - 3 # 32 kHz oscillator
+    enum: [0, 1, 2, 3]
+
+  battery-switchover:
+    type: int
+    default: 0
+    description: |
+      Enable the battery backup feature
+      the default is 0 (off)
+      - 0 # No battery backup
+      - 1 # Non-rechargeable battery backup i2c & FOUT are disabled when VDD is below Vdet1
+      - 2 # Non-rechargeable battery backup i2c & FOUT are always on
+      - 3 # Rechargeable battery backup i2c & FOUT are disabled when VDD is below Vdet1
+      - 4 # Rechargeable battery backup i2c & FOUT are always on
+    enum: [0, 1, 2, 3, 4]
```

Comment:
If there exist readible names for these enums, using a string would possibly be easier to read:
```
type: string
enum:
  - "no-backup"
  - "non-recharge-backup-no-fout"
  ...
```

Reply:
The battery backup feature dosn't have readable names in the data sheet.
I'm going to remove the 0 entry, but I'd rather keep the rest.